src/BlazorQuery.Library/BlazorQuery.Library.csproj

```diff
@@ -1,31 +1,38 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Razor">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <TargetFramework>netstandard2.1</TargetFramework>
+     <RazorLangVersion>3.0</RazorLangVersion>
     <AddRazorSupportForMvc>true</AddRazorSupportForMvc>
-    <AssemblyName>BlazorQuery</AssemblyName>
-    <RootNamespace>BlazorQuery</RootNamespace>
+    <AssemblyName>BlazorQuery.Library</AssemblyName>
+    <RootNamespace>BlazorQuery.Library</RootNamespace>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
     <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
-    <Version>0.0.1</Version>
-    <Authors>KevinJPetersen</Authors>
-    <RepositoryUrl>https://github.com/kevinjpetersen/BlazorQuery</RepositoryUrl>
-    <PackageProjectUrl>https://github.com/kevinjpetersen/BlazorQuery</PackageProjectUrl>
+    <Version>0.0.2</Version>
+    <Authors>Sybaris (Jean-Pierre Planas), KevinJPetersen</Authors>
```

Comment:
Same here in terms of Contributor Tags. I think it makes the most sense to just include the Repo Author. We can always add Contributors to the Description?

Reply:
Done. I set again thr url to your repo.